libs/com.etendorx.generate_entities/src/main/resources/org/openbravo/base/gen/baseDTORepository.ftl

```diff
@@ -113,20 +120,38 @@ public class BaseDTORepositoryDefault<T extends BaseSerializableObject,E extends
         throw new IllegalStateException("Entity conversion failed");
       }
       setAuditValuesIfApplicable(entity, isNew);
-      repository.save(entity);
-      repository.save(converter.convertOneToMany(dtoEntity, entity));
+      Set<ConstraintViolation<T>> violations = validator.validate(entity);
+      if (!violations.isEmpty()) {
+        List<String> messages = new ArrayList<>();
+        boolean hasViolations = false;
+        for (ConstraintViolation<T> violation : violations) {
```

Comment:
for (ConstraintViolation<T> violation : violations) {
          if (!StringUtils.equals(violation.getPropertyPath().toString(), "id")) {
            messages.add(violation.getPropertyPath() + ": " + violation.getMessage());
            hasViolations = true;
            break;
          }
        }